discord/embeds.py

```diff
@@ -405,7 +395,7 @@ def colour(self) -> MaybeEmpty[Colour]:
         return getattr(self, "_colour", EmptyEmbed)
 
     @colour.setter
-    def colour(self, value: Union[int, Colour, _EmptyEmbed]):  # type: ignore
+    def colour(self, value: int | Colour | _EmptyEmbed):  # type: ignore
```

Comment:
We're changing the base python version to 3.10?

Reply:
See https://discord.com/channels/881207955029110855/929023758742749244/1026859802808307742

Reply:
TL;DR: `from __future__ import annotations` allows this on python 3.8+